### ISSUE TITLE
is it okay to make a brew cask for it?

### ISSUE THREAD

**Issue by MoritzFago** [DouglasHeriot/PresenterMirror#4]
i want to make a home-brew cask https://github.com/caskroom/homebrew-cask for it.
Would you be okay with that? @DouglasHeriot

**Comment by DouglasHeriot** [comment]
Thanks for asking, but I’m thinking probably no. My original idea was it’d be nice to see if I could make some money from this as it’s useful for professionals giving presentations.

Some history:
The Creative Commons Non-commercial license on the source code came about because I originally used [v002 Media Tools](https://github.com/v002/v002-Media-Tools) screen capture Quartz Composer plugin, and it has a [CC-BY-NC-SA license](http://v002.info/licenses/).
To improve performance, I ditched Quartz Composer. The side-effect of that was I could charge money, as it no longer used any CC-NC code. I decided to keep it open source, as I have software-developer friends who find it interesting and useful.

However, I’ve been thinking about adding extra features that I would _not_ release openly (particularly around [Syphon](https://github.com/Syphon/Syphon-Framework) for monitoring pro live video software)

Technically, with the CC license, you’re allowed to redistribute it with attribution and the same license, so I suppose if you really wanted to you could make a cask. I’m curious what your plan is – if you would just find it useful in your personal Mac(s), or if it would be deployed across many machines.

In summary:
- If for private use only, you could [make your own cask tap](https://github.com/caskroom/homebrew-cask/blob/master/doc/alternate_cask_taps.md). Give it an alternate name, like "presenter-mirror-open", to avoid confusion from any future version I may release.
- If for wide-spread use, to avoid confusion I’d prefer maintaining it myself (and plan at some point to include my own licensing system)

Thanks.
